src/ecos/ecos.py

```diff
@@ -50,8 +50,8 @@ def solve(c,G,h,dims,A=None,b=None, **kwargs):
 
     if G is None:
         data = np.zeros((0,),dtype=np.double)
-        indices = np.zeros((0,),dtype=np.int)
-        colptr = np.zeros((n1+1,),dtype=np.int)
+        indices = np.zeros((0,),dtype=int)
+        colptr = np.zeros((n1+1,),dtype=int)
```

Comment:
In CVXPY, within the flattened tensor representation we had to use `np.int64` as windows uses 32 ints by default. We could use it here as well to be prudent. Just commit below if you think it is worthwhile. 
```suggestion
        indices = np.zeros((0,),dtype=np.int64)
        colptr = np.zeros((n1+1,),dtype=np.int64)
```

Reply:
Confirmed that using `int` works with all CVXPY tests.